components/01-atoms/forms/_form-element.twig

```diff
@@ -0,0 +1,88 @@
+{#
```

Comment:
This file in consistent with other projects we use but is very drupal-y. Not saying something needs to change here to merge this but it would be nice to figure out a different solution or something that is more inline with the other files. Just my two-cents :stuck_out_tongue:

Reply:
@callinmullaney Do you have ideas as to how to make it less Drupal-y? Being that we need form validation from Drupal, I'm not sure how we'd do forms differently.

Reply:
Does it make sense to move any of this to a template file in the atomic-theme? I know that our wiring-files are usually pretty superficial and most of the work is done in other places. But forms sit in that special place where the markup implies a lot of functionality.

Of if you want me to do any custom field logic in PHP, I can alter elements/attributes in the ys_theme module?

Reply:
@callinmullaney @nJim I suppose we could remove the `_form-element.twig` and `_form-label.twig` files in the component library and instead of `{% include`-ing them in the Drupal template, simply use the markup instead. What are your thoughts? That way at least it's not in the component library. 

We really just wanted a way to show what forms should look like in Storybook.